moved/src/move_execution/tests/utils.rs

```diff
@@ -1,4 +1,7 @@
-use {super::*, crate::types::transactions::NormalizedExtendedTxEnvelope};
+use {
+    super::*, crate::types::transactions::NormalizedExtendedTxEnvelope,
```

Comment:
I think `NormalizedExtendedTxEnvelope` got snuck in by accident before.
This should just be `use super::*;` and everything else would go into `mod.rs`.